### PR TITLE
[CDU] Duplicate waypoint names page rework

### LINKED ISSUE #1480: MCDU - Waypoint distance in the list of duplicate waypoints.
**Is your feature request related to a problem? Please describe.**
It is a request

**Describe the feature you'd like to see implemented**
When there are more waypoints with same names in the database, and you want to insert into the flight plan, the mcdu will give you a list of waypoints with gps coordinates. On a real MCDU on the right side the distance of the waypoint should be presented to that it is obvious to select the closest (proper) one.

**References**
https://www.youtube.com/watch?v=c7UJlJMJdfk&feature=youtu.be&t=591&ab_channel=320SimPilot
This guy is a real A320 pilot and clearly says that on the right side normally there is the distance from your position.

mcdubefore-1
mcduafter-1

**Additional context**
N/A

Discord username (if different from GitHub):
BreakerHUN#3986

### PR DESCRIPTION
**Pull request by pessip**
Fixes #1480

## Summary of Changes

I reworked the duplicate waypoint page to make it look more like the real one. I added distance to waypoint and navaid frequencies (on airports it just uses the first frequency in the frequencies array). I removed the waypoint type. Visually I redisigned the page with columns and added colours.

## Screenshots (if necessary)

Before:
before
After:
after

## Additional context

Normally the number of nautical miles is in green and NM in white but that doesn't seem to be possible with the sim.

Discord username: pessimoo#6055

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build.py will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, slick on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Comment by wpine215** [comment]
Is it possible to remove the "LAT/LONG" and "FREQ" titles for all but the top small-text row, to match the reference?

**Comment by pessip** [comment]
@wpine215 I'll do that

**Comment by MisterChocker** [comment]
Does the screenshot above shows the before or after version?

**Comment by pessip** [comment]
@MisterChocker I added a before screenshot. That was the after version

**Comment by MisterChocker** [comment]
Thanks, could you add the * in front of the indicator as well?

**Comment by MisterChocker** [comment]
Am I the only one seeing the L in the last row at ENO? Is this a bug or feature?

**Comment by pessip** [comment]
@MisterChocker Yeah I thought that was wierd also but I haven't changed anything related to that so it's probably just normal behaviour.

**Comment by MisterChocker** [comment]
seems odd, you could check the name tho before inserting them to the list.

**Comment by MisterChocker** [comment]
Would also be nice to remove LAT/LONG and FREQ for every row except the the most top.

**Comment by pessip** [comment]
I have confirmed that this works on the newest version and merged the changes from master.

**Comment by beheh** [comment]
After #1830 lands you should be able to split the label color here into green/white for distance/nm, as that does currently not match the reference.

**Comment by Benjozork** [comment]
Hello, please rework to fit #1830

**Comment by beheh** [comment]
#1830 has landed - after rebasing you can now do something like this:

```javascript
"{green}" + distance + "{end}nm"
```

...to get your split colors for e.g. `180nm`.

**Comment by beheh** [comment]
You should still implement sorting the waypoints by distance like in reality:

> The DUPLICATE NAMES page normally displays  the waypoints starting with the one closest to the aircraft.

image

The latter part ("is not displayed when two waypoints with the same identifier") could be handled in a different place as this page doesn't even need to load in that case, but also does not have to happen as part of this PR.

**Comment by pessip** [comment]
Great I'll work on trying to sort them today. And now that that works I van also add the colors!

**Comment by pessip** [comment]
What should be displayed if there is no frequency associated with the waypoint?

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU.html [hunk 1]
```diff
@@ -1,132 +1,132 @@
-﻿<link rel="stylesheet" href="A320_Neo_CDU.css" />
```
Comment (beheh) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU.html:1]: I think your commit messed up the line endings here. You probably will want to revert this file.

**Comment by wpine215** [comment]
Tested, LGTM!

**Pull request merged**